### PR TITLE
configured sidekiq server, and deploy cleanup

### LINKED ISSUE #467: sidekiq web ui admin link?
Introduce a link to /sidekiq for users who are administrators, probably in the users link dropdown in the upper right of the screen?

### PR DESCRIPTION
**Pull request by revgum**
fixes #467 

changes sidekiq address to something more generic for if/when sidekiq is replaced

### CODE REVIEW

File: app/views/_user_util_links.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<ul id="user_utility_links" class="nav navbar-nav navbar-right">
+  <% if user_signed_in? %>
+      <% if current_user.admin? %>
```
Comment (straleyb) [inline app/views/_user_util_links.html.erb]: Could bring this into one line with 
```
if user_signed_in? && current_user.admin?
```

**Comment by straleyb** [comment]
Looks good besides that one comment. 👍

File: app/views/_user_util_links.html.erb (hunk 1 above)
Comment (revgum) [inline app/views/_user_util_links.html.erb]: no, actually these are two separate functionalities.. only admin's should see the link, while all signed in users should see the other stuff as well

Comment (straleyb) [inline app/views/_user_util_links.html.erb]: Ahh ok yeah I see. Cool dude. Merging.

**Pull request merged**